src/screens/TopicPageScreen/index.js

```diff
@@ -421,6 +430,9 @@ const TopicPageScreen = (props) => {
         setIsFollow={setIsFollow}
         isFollow={isFollow}
         getTopicDetail={getTopicDetail}
+        onFollowButtonPress={() => bottomSheetFollowRef?.current?.open()}
+        followType={followType}
+        setFollowType={setFollowType}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `NavHeader` component now accepts new props related to following. Ensure these props are used correctly in the `NavHeader` component.
```diff
+ onFollowButtonPress={() => bottomSheetFollowRef?.current?.open()}
+ followType={followType}
+ setFollowType={setFollowType}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->